extra/completions/_alacritty

```diff
@@ -18,7 +15,6 @@ _arguments \
   "--embed=[define the X11 window ID (as a decimal integer) to embed Alacritty within]:windowId" \
   "(-e --command)"{-e,--command}"[execute command (must be last arg)]:program: _command_names -e:*::program arguments: _normal" \
   "--config-file=[specify an alternative config file]:file:_files" \
-  "(-d --dimensions)"{-d,--dimensions}"[specify window dimensions]:columns: :lines" \
-  "--position[specify window position]:x position: :y position" \
+  "(-o --option)"{-o+,--option=}"[override config file options (example: cursor.style=Beam)]:option" \
```

Comment:
It looks like examples/defaults are not provided in any other completion documentation, which makes some sense. So I'd advocate for removing it here too. So just `override config file options` should be sufficient.